frontend/src/app/app.component.less

```diff
@@ -13,6 +13,11 @@
   justify-content: center;
 }
 
+.hr {
+  margin-bottom: 3px;
+}
+
+
```

Comment:
I think there is one empty line too many

Reply:
Fixed